endorsement/models/shared_drive.py

```diff
@@ -184,33 +215,52 @@ class SharedDriveRecord(
 
     @property
     def expiration_date(self):
-        lifespan = getattr(settings, 'SHARED_DRIVE_LIFESPAN_DAYS', 365)
-        claim_span = getattr(settings, 'SHARED_DRIVE_CLAIM_DAYS', 30)
-        initial_date = (self.datetime_renewed or self.datetime_accepted)
-        return (self.datetime_expired or
-                (initial_date + timezone.timedelta(days=lifespan)) if (
-                    initial_date) else (
-                        self.datetime_created + timezone.timedelta(
-                            claim_span)) if (
-                                self.datetime_created) else None)
+        lifespan = getattr(settings, "SHARED_DRIVE_LIFESPAN_DAYS", 365)
+        claim_span = getattr(settings, "SHARED_DRIVE_CLAIM_DAYS", 30)
+        initial_date = self.datetime_renewed or self.datetime_accepted
+        return (
+            self.datetime_expired
+            or (initial_date + timezone.timedelta(days=lifespan))
+            if (initial_date)
+            else (
+                (self.datetime_created + timezone.timedelta(claim_span))
+                if (self.datetime_created)
+                else None
+            )
+        )
+
+    def get_itbill_key_remote(self):
+        """
+        Compute 'key_remote' field for ITBill.
+        """
+        # if changing be sure this does not exceed the max_length of
+        # ITBillSubscription.key_remote
+        key_remote = f"prt_sd_{self.shared_drive.drive_id}"
```

Comment:
I couldn't find any notes where we agreed on a prefix with the ITBill team. Do you have anything?